storage/db-core/src/main/resources/db-core.yml

```
@@ -6,13 +6,29 @@ spring:
     properties:
       hibernate.default_batch_fetch_size: 100
 
+mybatis:
+  # XML 매퍼 파일의 위치를 지정합니다. (매우 중요)
+  # 클래스패스 기준으로 여러 경로를 지정할 수 있습니다.
+  mapper-locations: classpath:/mappers/**/*.xml # src/main/resources/mappers 폴더 아래 모든 xml 파일
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)